src/main/java/core/basesyntax/AreaCalculator.java

```diff
@@ -0,0 +1,5 @@
+package core.basesyntax;
+
+public interface AreaCalculator {
+    public double calculateArea();

```

Comment:
it is redundant to use the access modifier with methods in the interfaces because all abstract methods by default are public

https://docs.oracle.com/javase/tutorial/java/IandI/interfaceDef.html#:~:text=All%20abstract%2C%20default%2C%20and%20static%20methods%20in%20an%20interface%20are%20implicitly%20public%2C%20so%20you%20can%20omit%20the%20public%20modifier.